hw/ip/tlul/data/tlul.prj.hjson

```diff
@@ -18,7 +21,7 @@
         life_stage:         "L1",
         design_stage:       "D1",
         verification_stage: "V1",
-        commit_id:          "2366287ed15fd02c09f05ce61c5d9ef2ae280181",
+        // commit_id:          "2366287ed15fd02c09f05ce61c5d9ef2ae280181",
         notes:              "Rolled back to D1V1 as more features (like multi-clock) are being updated"
```

Comment:
I recommend we remove this note. Notes should be pertaining to present
state not past history, as the history will fade. For instance it would be
appropriate to have a note as to why a new version exists for a signed-off
version, but otherwise the note should simply be in the revision history of
the checklist file PR.

Reply:
Done.